libs/watt/package/shell/nav-list/watt-expand-on-active-link.directive.ts

```diff
@@ -16,32 +16,37 @@
  * limitations under the License.
  */
 //#endregion
-import { Directive, AfterViewInit, inject, input } from '@angular/core';
+import { Directive, inject, input, effect, OutputRefSubscription } from '@angular/core';
 import { MatExpansionPanel } from '@angular/material/expansion';
-import { outputToObservable } from '@angular/core/rxjs-interop';
-import { from, filter, mergeMap } from 'rxjs';
 
-import { WattNavListItemComponent } from './watt-nav-list-item.component';
+import { type WattNavListItemComponent } from './watt-nav-list-item.component';
 
 @Directive({
   selector: '[wattExpandOnActiveLink]',
   exportAs: 'wattExpandOnActiveLink',
 })
-export class WattExpandOnActiveLinkDirective implements AfterViewInit {
+export class WattExpandOnActiveLinkDirective {
   private panel = inject(MatExpansionPanel);
 
-  wattNavListItemComponents = input<readonly WattNavListItemComponent[]>([]);
+  private subs: OutputRefSubscription[] = [];
```

Comment:
suggestion: make this a local variable in `navListItemsEffect`, since it doesn't look like it is being used anywhere else.

Reply:
Fixed https://github.com/Energinet-DataHub/greenforce-frontend/pull/4623/commits/0576560a9cc6277b0f7cd1600857c1c54da37db5